crates/egui_demo_lib/Cargo.toml

```diff
@@ -50,6 +50,9 @@ chrono = { version = "0.4", optional = true, features = ["js-sys", "wasmbind"] }
 ## Enable this when generating docs.
 document-features = { workspace = true, optional = true }
 serde = { version = "1", optional = true, features = ["derive"] }
+once_cell = "1.19"
+colorgrad = "0.6"
+noise = "0.9"
```

Comment:
No please don't - we already have enough dependencies to compile as it is.

And you can already parse hex colors: https://docs.rs/ecolor/latest/ecolor/macro.hex_color.html

Reply:
I can remove the once_cell and colorgrad dependencies, but unless you want me to throw together a noise function for the color demo, I'd like to keep the noise crate in. If that's too much, I can see what I can do.

Reply:
Just use a sine wave or something?

Reply:
I'll roll it in now, for some reason color_hex isn't resolving and I have no idea why, maybe it's a silly mistake I made

Reply:
Have you enabled the `color-hex` in `ecolor`?

Reply:
I've tried enabling it in both `egui` and `ecolor`, for some reason it's just refusing to include the color_hex dependency. I think it's a bug on my end, so I'm asking someone else to see if it runs on their machine.

Reply:
This was shaping up to be a massive mystery, until I saw #4372. Updating my branch fixed the issue. I did have to add back once_cell because color_hex!() can't be used in consts, maybe another PR is in order?

Reply:
`once_cell` is fine